databases/week2/part4erdiagram1.sql

```diff
@@ -0,0 +1,48 @@
+/*CREATE TABLE `Staff` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,  
+  `email` varchar(255) NOT NULL,
+  `phone` varchar(255) NULL, 
+  `office_id` int(10) NOT NULL,
+  `department_id` int(10) NOT NULL,
```

Comment:
Looks like you also tried to reverse the relations by adding these two columns. Yet you don't have foreign key constraints for the relations.